build.yaml

```diff
@@ -1677,6 +1677,24 @@ libs:
   - grpc_test_util
   - grpc
   - gpr
+- name: callback_test_service
```

Comment:
I don't think that this is a good name for it in build.yaml since this is non-hierarchical. I'd prefer bm_callback_test_service at least